sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/StorageException.java

```diff
@@ -19,34 +20,29 @@
  */
 public final class StorageException extends HttpResponseException {
     private static final String ERROR_CODE = "x-ms-error-code";
```

Comment:
You can use the Constant in common instead of this local String.

Reply:
Done.